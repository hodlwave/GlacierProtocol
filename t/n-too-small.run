#!/bin/bash

$PROOFWALLET view-addresses -m 2 -n 1 --regtest << INPUT 2>&1
INPUT

retVal=$?
if [ $retVal -eq 0 ]; then
    echo "Error: expected proofwallet.py to fail"
    exit 1
fi
