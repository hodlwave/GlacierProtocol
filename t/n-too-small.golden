Error: n must be between 2 and 15
